roles/bindle-profiles/tasks/main.yml

```diff
@@ -16,26 +16,31 @@
   git: repo=https://github.com/CloudBindle/Bindle
        dest=/home/{{ user_name }}/architecture2/Bindle
        version={{ bindle_git_branch }}
+       force=no
```

Comment:
When a file in any of the repos (in this case, pancancer-bag) is modified, the exact error message is:
<pre>
TASK: [bindle-profiles | Checkout the Bindle git repo] ************************ 
ok: [bindle]

TASK: [bindle-profiles | Checkout the Seqware git repo] *********************** 
ok: [bindle]

TASK: [bindle-profiles | Checkout the pancancer-bag git repo] ***************** 
failed: [bindle] => {"failed": true}
msg: Local modifications exist in repository (force=no).

FATAL: all hosts have already failed -- aborting

PLAY RECAP ******************************************************************** 
           to retry, use: --limit @/home/ubuntu/site.retry

bindle                     : ok=18   changed=5    unreachable=0    failed=1   

ubuntu@ip-10-111-186-42:~/architecture-setup$ 
</pre>

I'm not sure if there's any way to get a clearer message from git (such as the output of `git status`) but this should at least indicate to the user that they need to fix that repo before they can move on.